src/redux/data.ts

```diff
@@ -1396,10 +1395,18 @@ export const dataWatchPendingTransactions = (
             txObj
           );
 
-          if (updatedPendingTransaction?.swap?.type === SwapType.crossChain) {
+          // approvals are not via socket so we dont want to check their status with them.
+          const isApproveTx =
+            transactionStatus === TransactionStatus.approved ||
+            transactionStatus === TransactionStatus.approving;
+          if (
+            updatedPendingTransaction?.swap?.type === SwapType.crossChain &&
+            !isApproveTx
+          ) {
             pendingTransactionData = await getTransactionSocketStatus(
               updatedPendingTransaction
             );
+            logger.debug('pending tx data: ', pendingTransactionData);
```

Comment:
switch to new logger?

Reply:
ah yes I was going to but got scared with all the obj we're logging, will revisit

Reply:
Yeah this seems like an OK `debug`. Not sure we need this in Sentry, but might be useful locally 👍

Reply:
Object is correct in this case, it'll just be pretty-printed to the console.